templates/download/amd-xilinx/index.html

```diff
@@ -49,16 +52,9 @@ <h1>Install Ubuntu on Xilinx</h1>
             <h2>Download Ubuntu Desktop</h2>
             <h3 class="p-heading--4">Ubuntu Desktop 22.04 LTS</h3>
             <p class="u-sv1">The version of Ubuntu with up to 10 years of long term support, until April 2032.</p>
-            <div class="p-notification--information">
-              <div class="p-notification__content">
-                <h5 class="p-notification__title">Note:</h5>
-                <p class="p-notification__message">This is a public Beta release for KR260 with limited support (see <a href="https://support.xilinx.com/s/question/0D52E00007BslVuSAJ/ubuntu-2204-lts-beta-for-kria?language=en_USMPSAY/certified-ubuntu-2004-lts-for-xilinx-devices?language=en_US">this forum post</a> for details); please check back for a full production release image in Q3 2022.
-                </p>
-              </div>
-            </div>
             <p>
               <a class="p-button--positive"
-                 href="https://people.canonical.com/~platform/images/xilinx/kria-ubuntu-22.04/iot-limerick-kria-classic-desktop-2204-x04-20220517-68.img.xz">Download 64-bit</a>
+                 href="https://people.canonical.com/~platform/images/xilinx/kria-ubuntu-22.04/iot-limerick-kria-classic-desktop-2204-x06-20220614-78.img.xz">Download 64-bit</a>
             </p>
             <p>Works on:</p>
             <ul class="p-list">
```

Comment:
It won't let me comment on them directly, but the list items below need the "(beta)" removed

Reply:
Ah nice, updated